lib/adobexd.js

```diff
@@ -23,6 +23,13 @@ async function getArtboardFiles(outputDir) {
   return artboardObjects;
 }
 
+async function getTextStyles(outputDir) {
+  const source = path.join(outputDir, 'resources', 'graphics', 'graphicContent.agc');
+  const { documentLibrary } = JSON.parse(await readFile(source)).resources.meta.ux;
+
+  return documentLibrary.elements;
```

Comment:
To be sure only to include text styles I would add an extra filter here:

```suggestion
  return Array.isArray(documentLibrary.elements) ? documentLibrary.elements.filter(e => e.type.startsWith('application/vnd.adobe.element.characterstyle')) : undefined;
```

Reply:
How should it behave if there are no textstyles defined in the file? I was thinking of returning an empty array so it doesn't crash the flow of the program.

Reply:
that's fine